eisbuk-admin/src/components/atoms/BirthdayMenu/BirthdayMenu.tsx

```diff
@@ -37,6 +38,17 @@ const BirthdayMenu: React.FC<Props> = ({ customers, onClickShowAll }) => {
     setBirthdaysAnchorEl(null);
   };
 
+  const calculateTimeDiff =
```

Comment:
`calculateTimeDiff` evokes a function, since it starts with a verb. A nou would be more appropriate, to describe the value contained here. `millisToMidnight` IMO would be more descriptive.